src/sentry/issues/priority.py

```diff
@@ -70,6 +77,34 @@ def update_priority(
     if group.type == MetricIssue.type_id:
         update_incident_activity_based_on_group_activity(group, priority)
 
+    # create a row in the GroupOpenPeriodActivity table
+    open_period = get_latest_open_period(group)
+    if open_period is None:
+        logger.error("No open period found for group", extra={"group_id": group.id})
+    else:
+        if get_group_type_by_type_id(group.type).track_priority_changes:
+            if is_regression:
+                try:
+                    activity_entry_to_update = GroupOpenPeriodActivity.objects.get(
+                        group_open_period=open_period, type=OpenPeriodActivityType.OPENED
+                    )
+                    activity_entry_to_update.update(value=priority)
+                except GroupOpenPeriodActivity.DoesNotExist:
+                    # in case the rollout somehow goes out between open period creation and priority update
```

Comment:
thoughts on sending a metric here? i'd be curious to see how frequently this is happening since we don't expect it to happen often.